### PR TITLE
refactor!(server): drop Cassandra support

### PR DESCRIPTION
**Pull request by syncush**
As discussed with @NivLipetz, this PR drops predator's Cassandra support

### CODE REVIEW

File: src/configManager/models/database/databaseConnector.js [hunk 1]
```diff
@@ -1,10 +1,7 @@
 'use strict';
-
-const databaseConfig = require('../../../config/databaseConfig');
-const cassandraConnector = require('./cassandra/cassandraConnector');
 const sequelizeConnector = require('./sequelize/sequelizeConnector');
```
Comment (enudler) [inline src/configManager/models/database/databaseConnector.js:2]: can be changed to databaseConnector

File: src/database/database.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
 'use strict';
 
-let databaseConfig = require('../config/databaseConfig');
-let cassandraConnector = require('./cassandra-handler/cassandra');
 let sequelizeConnector = require('./sequlize-handler/sequlize');
-let databaseConnector = databaseConfig.type.toLowerCase() === 'cassandra' ? cassandraConnector : sequelizeConnector;
+let databaseConnector = sequelizeConnector;
```
Comment (enudler) [inline src/database/database.js:4]: sequelizeConnector can be changed to databaseConnector

**Comment by enudler** [comment]
@NivLipetz @syncush 
LGTM

We need to update our readme from now that 1.5 dropping Cassandra support. and master will drop support for some specific date.

**Comment by syncush** [comment]
Would you like me to create a pr?

**Comment by enudler** [comment]
Would be great
thanks @syncush

File: src/configManager/models/database/databaseConnector.js (hunk 1 above)
Comment (syncush) [reply src/configManager/models/database/databaseConnector.js]: done

File: src/database/database.js (hunk 2 above)
Comment (syncush) [reply src/database/database.js]: done

File: tests/unit-tests/jobs/models/jobManager-test.js [hunk 3]
```diff
@@ -1270,4 +1270,8 @@ describe('Manager tests', function () {
             }
         });
     });
+
```
Comment (NivLipetz) [inline tests/unit-tests/jobs/models/jobManager-test.js:1273]: think you can remove ;)

File: tests/unit-tests/webhooks/sequelize/webhooksFormatter-test.js [hunk 4]
```diff
@@ -0,0 +1,233 @@
+const sinon = require('sinon');
```
Comment (NivLipetz) [inline tests/unit-tests/webhooks/sequelize/webhooksFormatter-test.js:1]: can remove file, this file is in master under
`unit-tests/webhooks/webhooksformatter-test.js`

**Pull request merged**